packages/backend/src/database/pagination/errors.ts

```diff
@@ -0,0 +1,6 @@
+export class KnexPaginationError extends Error {
```

Comment:
why not move it inside `common` `errors` next to the other error types , and `extends LightdashError`